### DISCUSSION TITLE
Android Support?

### DISCUSSION THREAD

**Discussion by ksant0s** [yokoffing/Betterfox#223] [Q&A] [answered]
Firefox Beta and Nightly version support about:config in android. Changes made to config section are  stored in pref.js found in `/data/data/<package name>/files/mozilla/<profile>.default/` . Is it safe to enable flags from yours user.js in android too or these are specially for desktop version of Firefox?

**Comment by yokoffing** [comment, marked as answer]
Unofficially, yes. See also https://github.com/yokoffing/Betterfox/issues/143, if applicable.

**Reply by yokoffing** [reply]
@ksant0s You can try and test this, and let me know how it fairs for you: https://github.com/yokoffing/Betterfox/issues/240
